src/index.js

```diff
@@ -32,8 +32,8 @@ export default declare(({
     if (typeof importPath !== 'string') {
       throw new TypeError('`applyPlugin` `importPath` must be a string');
     }
-    const { ignorePattern, caseSensitive } = state.opts;
-    const { file } = state;
+    const { ignorePattern, caseSensitive, filename: providedFilename } = state.opts;
+    const { file, filename } = state;
```

Comment:
Looks like `filename` was originally coming from `state.file.opts` on ln 47, was this updated somewhere?

Reply:
I'm no longer using that value, but it seems like it will be populated by `state.filename` when transforming a file, and `state.opts.filename` when the option is provided, so this seemed more straightforward.